eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/config/oauth2/HttpCookieOAuth2AuthorizationRequestRepository.java

```diff
@@ -0,0 +1,55 @@
+package hu.flowacademy.meetingorganizer.config.oauth2;
+
+import com.nimbusds.oauth2.sdk.util.StringUtils;
+import hu.flowacademy.meetingorganizer.utils.CookieUtils;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.oauth2.client.web.AuthorizationRequestRepository;
+import org.springframework.security.oauth2.core.endpoint.OAuth2AuthorizationRequest;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+public class HttpCookieOAuth2AuthorizationRequestRepository implements AuthorizationRequestRepository<OAuth2AuthorizationRequest> {
+    public static final String OAUTH2_AUTHORIZATION_REQUEST_COOKIE_NAME = "oauth2_auth_request";
+    public static final String REDIRECT_URI_PARAM_COOKIE_NAME = "redirect_uri";
+    public static final String ALTERNATIVE_REDIRECT_URI = "http://localhost:4200/auth";
```

Comment:
Ez is jöhetne yaml-ből

Reply:
Valamiért erre hibát dob nekem: ugyanúgy felviszem yaml-be: 
auth:
  OAUTH2_AUTHORIZATION_REQUEST_COOKIE_NAME: oauth2_auth_request
  REDIRECT_URI_PARAM_COOKIE_NAME: redirect_uri
  ALTERNATIVE_REDIRECT_URI: http://localhost:4200/auth
  cookieExpireSeconds: 180
Csinálok egy AuthConfigs-ot, de nem engedi átadni a paramétereket belőle és hibát dob. Lehet ezt egy későbbi issue-ban megpróbálnám külön megoldani.

Reply:
Arra gondoltam, hogy elég lenne, ha csak kulcs alapján betöltenéd `@Value`-val egy változóba.